.travis.yml

```diff
@@ -65,6 +65,7 @@ matrix:
     - language: python
       python: "3.6"
       env: TOXENV=py36
+      install: pip3 install jsonschema==3.0.1
```

Comment:
Hmm. This should not be needed as you have jsonschema in setup.py

Reply:
Synced offline. Due to the travis system, this package needs to be installed using pip3.